openassessment/templates/openassessmentblock/response/oa_response_cancelled.html

```diff
@@ -24,26 +24,17 @@
                 <h3 class="message__title">{% trans "Submission Cancelled" %}</h3>
 
                 <div class="message__content">
-                  <p>
-                    {% blocktrans with removed_datetime=workflow_cancellation.created_at|utc|date:"N j, Y H:i e" removed_by_username=workflow_cancellation.cancelled_by %}
-                        Your submission has been cancelled by {{ removed_by_username }} on {{ removed_datetime }}
-                    {% endblocktrans %}
-                    <br>
-                    <!-- Comments: Reason for Cancellation-->
-                    {% blocktrans with comments=workflow_cancellation.comments %}
-                        Comments: {{ comments }}
-                    {% endblocktrans %}
-                  </p>
-                </div>
-
-                <div class="step__content">
-                    <article class="submission__answer__display">
-                        <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
-
-                        <div class="submission__answer__display__content">
-                            {{ student_submission.answer.text|linebreaks }}
-                        </div>
-                    </article>
+                    <p>
+                        {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|utc|date:"N j, Y H:i e" removed_by_username=workflow_cancellation.cancelled_by %}
```

Comment:
@andy-armstrong Not sure if this is going to require a separate story (or stories), but working on XSS presentation with Ned, we realized a couple of things:
1. Where we use trans and blocktrans we don't seem to be escaping the translation, so we need to establish a best practice.
2. The [i18n wiki page](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html) doesn't include Django templates
3. The way we are handling "blocktrans" doesn't seem like it would work right for translations, and also might introduce double escaping.  Either way, the best practice should be documented in the wiki when it is established.
Thanks!